src/standalone.js

```diff
@@ -231,6 +231,17 @@ export default async function(initialOptions) {
   const result = {};
 
   result.glyphsData = await getGlyphsData(filteredFiles, options);
+
+  if (options.glyphTransformFn) {
+    const transformedGlyphs = result.glyphsData.map(async glyphData => {
+      const metadata = await options.glyphTransformFn(glyphData.metadata);
+
+      return { ...glyphData, metadata };
+    });
+
+    result.glyphsData = await Promise.all(transformedGlyphs);
```

Comment:
We don't need `await Promise.all` here, `await options.glyphTransformFn(glyphData.metadata)` already return changed data

Reply:
not completely as this map function is also async we get a list of promises that are not resolved.
[example](https://jsfiddle.net/0rpudyob/1/)

Reply:
:+1: Let's add test for this case too

Reply:
do you mean the case that the `glyphTransformFn` function throws an error/rejects?

Reply:
Yes

Reply:
@Narquadah do we have any news on this part?